packages/google_maps_flutter/google_maps_flutter_ios/lib/src/google_map_inspector_ios.dart

```diff
@@ -103,6 +104,60 @@ class GoogleMapsInspectorIOS extends GoogleMapsInspectorPlatform {
     );
   }
 
+  @override
+  bool supportsGettingGroundOverlayInfo() => true;
+
+  @override
+  Future<GroundOverlay?> getGroundOverlayInfo(GroundOverlayId groundOverlayId,
+      {required int mapId}) async {
+    final PlatformGroundOverlay? groundOverlayInfo =
+        await _inspectorProvider(mapId)!
+            .getGroundOverlayInfo(groundOverlayId.value);
+
+    if (groundOverlayInfo == null) {
+      return null;
+    }
+
+    // Create dummy image to represent the image of the ground overlay.
+    final BytesMapBitmap dummyImage = BytesMapBitmap(
+      Uint8List.fromList(<int>[0]),
+      bitmapScaling: MapBitmapScaling.none,
+    );
+
+    if (groundOverlayInfo.position != null) {
```

Comment:
Nit: If you make locals for `groundOverlay.position` and `groundOverlay.bounds` just before the if block, and use the locals everywhere after that (including the conditions), you won't need most of the force-unwraps below.